.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+
+on:
+  workflow_call:
+  workflow_dispatch:
```

Comment:
Since the previous monolithic workflow had this option, I included it here. However, you're right—it doesn't seem to be very useful, particularly because this workflow is a dependency of others and there's no reason to call it on its own.